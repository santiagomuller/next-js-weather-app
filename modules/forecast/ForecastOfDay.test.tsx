import React from 'react';
import renderer from 'react-test-renderer';
import ForecastOfDayItem from './ForecastOfDay';

const mockWeatherData = {
  weather: [{
    icon: 'sarasa',
    description: 'icon',
  }],
  temp: { min: 10, max: 20 }
}

it('ForecastOfDayItem renders without changes', () => {
  const tree = renderer.create(
    <ForecastOfDayItem data={mockWeatherData} currentDay="Lunes" />
  ).toJSON();
  
  expect(tree).toMatchSnapshot();
});