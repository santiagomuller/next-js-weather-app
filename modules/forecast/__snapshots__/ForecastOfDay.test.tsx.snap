// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ForecastOfDayItem renders without changes 1`] = `
<div
  className="flex justify-between p-4 text-indigo-700 items-center"
>
  <p
    className="family-bold font-bold  w-20 pr-2"
  >
    Lunes
  </p>
  <div
    className="family-bold font-bold"
  >
    <img
      alt="icon"
      src="http://openweathermap.org/img/wn/sarasa.png"
    />
  </div>
  <div
    className="family-bold font-bold"
  >
    20
    ° - 
    10
    °
  </div>
</div>
`;
